gems/openssl/CVE-2017-14033.yml

```diff
@@ -0,0 +1,32 @@
+---
+gem: openssl
+cve: 2017-14033
+ghsa: v6rp-3r3v-hf4p
+url: https://www.ruby-lang.org/en/news/2017/09/14/openssl-asn1-buffer-underrun-cve-2017-14033/
+title: Ruby OpenSSL DoS Vulnerability
+date: 2022-05-14
+description: |
+  The decode method in the OpenSSL::ASN1 module in Ruby before 2.2.8, 2.3.x
+  before 2.3.5, and 2.4.x through 2.4.1 allows attackers to cause a denial of service
+  (interpreter crash) via a crafted string.
+cvss_v2: 5.0
+cvss_v3: 7.5
+patched_versions:
+  - "> 2.0.0"
```

Comment:
Can you confirm that [openssl-2.0.0](https://github.com/ruby/openssl/blob/v2.0.0/History.md) patches this?

Reply:
I got the "2.0.0" data from the url: reference:
```
If you can’t upgrade Ruby itself, install OpenSSL gem newer than version 2.0.0
```